libsawtooth/src/publisher/context.rs

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+//! Contains the trait for the context used by the publisher
+
+/// The context used by the publisher
+pub trait PublisherContext: Send {}
```

Comment:
`PublishingContext` might make more sense, as it’s a context for the process of publishing.

Reply:
Updated name